recipes/vibration.py

```diff
@@ -86,15 +91,16 @@ def new_children(self):
     def postrun(self):
         #get gradients and save them to the pickle files for ASE
         path = self.path
+        forces = {}
         for child in self.children:
             res = child.results
             name = child.name
             # don't rely on gradients beeing numpy arrays
-            f = [ npa(vec) for vec in self.get_grad(res, eUnit='eV', lUnit='Angstrom') ]
-            force = -1.0 * npa(self.reorder(res, f))
-            filename = osPJ(path, name+'.pckl')
-            with open(filename, 'wb') as f:
-                pickleDump(force, f, protocol=2)
+            f = [ npa(vec) for vec in self.get_grad(res, energy_unit='eV', dist_unit='Angstrom') ]
+            forces[name+'.pckl'] = -1.0 * npa(self.reorder(res, f))
+        filename = osPJ(path, self.name+'.all.pckl')
+        with open(filename, 'wb') as f:
+            pickleDump(forces, f, protocol=2)
```

Comment:
Is there a specific reason for using `protocol=2` here?
`protocol=3` has been the default since python 3.0; in python >3.8 it's been upped to `protocol=4` in fact ([docs](https://docs.python.org/3/library/pickle.html#data-stream-format)).

Reply:
The reason is that it is there in ASE... Why this is the case: No clue. I'll ask there.

Reply:
Seems fair; I would say that adding a comment for future reference would be a good idea though.

Reply:
I can even give you one better:https://github.com/patrickmelix/PLAMS/tree/vibration-json
Nothing done yet, but I checked how it looks in ASE and the interface should become even easier.